apina-core/src/main/kotlin/fi/evident/apina/output/ts/AbstractTypeScriptGenerator.kt

```diff
@@ -208,7 +209,7 @@ abstract class AbstractTypeScriptGenerator(
         val signature = "${endpoint.name}($parameters): $resultFunctor<$resultType>"
```

Comment:
Method parameters could be automatically expanded to always allow both `null` and `undefined`. So the parameter type could become `Foo | null | undefined` regardless of translation-settings. Furthermore, we could improve the code so that trailing nullable parameters would become optional. So we could have a signature like:

```typescript
foo(x: number | null | undefined, y: number, z?: number | null, w?: number | null): ...
```

But I understand that I'm augmenting the scope of the PR right now. So there's no need to do this, I'm just thinking out loud here. Of course if someone would submit a PR doing this, I'd happily merge that as well... (Just yesterday some colleagues talked about this.)

Reply:
I guess that's true yeah. I would by happy to just use optional props in method parameters and maybe leave the null possibility for later on if necessary? :)

Reply:
Well I applied your suggestion and now it's like this. :)